src/components/BoardList/BoardList.component.js

```diff
@@ -18,33 +23,81 @@ const BoardList = () => {
             };
             getBoardData();
         } catch (error) {
-            console.log(error.message);
+            console.log(error);
         }
     }, []);
 
     const buttonProperties = {
         text: "New Board",
     };
 
-    const addHandler = () => {
-        const newData = {
+    const autoFocus = () => document.querySelector("input").focus();
+
+    const handleCreate = () => {
+        setCreated(!created);
+    };
+
+    const handleSave = () => {
+        const boardName = text;
+        const date = String(new Date().toLocaleDateString());
+        let newData = {
             id: boards.length + 1,
```

Comment:
Board ids should be unique.
Think about a situation when we will have delete board functionality and you delete not the last item
What will happen then?
Think about a solution